dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/state/ApplicationStateListenerConstants.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2011 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.container.service.state;
+
+public class ApplicationStateListenerConstants {
+
+    //This class stores all the service rankings for application state listeners in one place alongside the reasons for them to have that rank.
+
+    public static final int DEFAULT_SERVICE_RANK = 0;
+
+    //CDI must shut down after EJB as EJBs can have a cdi application scope and application scoped beans must be shut down before CDI sends
+    //jakarta.enterprise.event.Shutdown
```

Comment:
This comment says beans are destroyed before shutdown, the test seems to want beans destroyed after shutdown.